_posts/pl/2024-10-11-czy-wiesz-ze-od-angular-17-mozna-uzywac-if-zamiast-ngif.md

```diff
@@ -0,0 +1,50 @@
+---
+layout:    post
+title:     Czy wiesz, że w Angular 17 została wprowadzona alternatywa dla *ngIf?
+description: ""
+date:      2024-10-11T08:00:00+01:00
+published: true
+didyouknow: true
+lang: pl
+author: dmejer
+image: /assets/img/posts/2024-10-11-czy-wiesz-ze-od-angular-17-mozna-uzywac-if-zamiast-ngif/thumbnail.webp
+tags:
+- javascript
+---
+W Angular 17 pojawiło się *built-it control flow*, a w 18 zostało ustabilizowane. Są to zamienniki dyrektyw `*ngIf`, `*ngFor`, `*ngSwitch`.
```

Comment:
W Angular 17 pojawiło się *built-it control flow*, które zostało ustabilizowane w wersji 18. Są to zamienniki dyrektyw `*ngIf`, `*ngFor` i `*ngSwitch`.